src/main/java/com/hubspot/jinjava/tree/parse/TokenScannerSymbols.java

```diff
@@ -17,13 +17,22 @@
 
 public interface TokenScannerSymbols {
 
-  int TOKEN_PREFIX = '{';
-  int TOKEN_POSTFIX = '}';
-  int TOKEN_FIXED = 0;
-  int TOKEN_NOTE = '#';
-  int TOKEN_TAG = '%';
-  int TOKEN_EXPR_START = '{';
-  int TOKEN_EXPR_END = '}';
-  int TOKEN_NEWLINE = '\n';
+  char TOKEN_PREFIX_CHAR = '{';
```

Comment:
for convenience it might be nice to have constants for `{%`, `%}`, `{{` and `}}`